content/hi/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें."
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
```

Comment:
```suggestion
[क्यूबकॉन्फिग फाइल](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)(kubeconfig) की आवश्यकता होती है,
```

Reply:
this makes things more clear! we want readers to know how to pronounce it but we also want them to know this is a `kubernetes object` and how it looks in English